server/planning/content_api/events/event.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2025 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+from superdesk.core.resources import ModelWithVersions
+from planning.types import EventResourceModel
+from superdesk.core.resources import fields
+from superdesk.types.base import CVItem
+from pydantic import Field
+from typing import Any
+
+
+class ContentAPIEventResourceModel(EventResourceModel, ModelWithVersions):
+    headline: fields.HTML | None = None
+    description_text: str | None = None
```

Comment:
Missing re-defining fields that will be expanded. Please look at [event formatter](https://github.com/superdesk/superdesk-planning/blob/async/server/planning/output_formatters/json_event.py) and [planning formatter](https://github.com/superdesk/superdesk-planning/blob/async/server/planning/output_formatters/json_planning.py) for an example of the fields we expand on.

At the moment some things are stored with just it's ID. We should instead be expanding on them, just like we do when we publish them to Newshub.

So please go over those formatters thoroughly, and find out what fields are expanded and update this and planning schema to reflect